projects/fsastorefront-e2e-cypress/cypress.json

```diff
@@ -1,12 +1,13 @@
 {
-  "defaultCommandTimeout": 10000,
-  "baseUrl": "http://10.27.246.242",
+  "defaultCommandTimeout": 20000,
```

Comment:
This should stay as is. We have separate configurations for local and test server. You can create more separate configurations if needed and call them via CLI.

Reply:
we should not commit our test environments